test/login.go

```diff
@@ -116,6 +116,6 @@ type DummyOSORegistrationApp struct {
 	Err    error
 }
 
-func (regApp *DummyOSORegistrationApp) LoadOSOSubscriptionStatus(ctx context.Context, config login.Configuration, keycloakToken oauth2.Token) (string, error) {
+func (regApp *DummyOSORegistrationApp) LoadOSOSubscriptionStatus(ctx context.Context, config login.Configuration, OAuth2Token oauth2.Token) (string, error) {
```

Comment:
this variable's name should start with a lowercase (ie, `oauth2Token`), or even just `t`, as short names are usually preferred